site/docs/glossary.md

```diff
@@ -0,0 +1,537 @@
+conjunction with an actual remote blob store.
+
+#### Distdir
+
+A read-only directory containing files that Bazel would otherwise fetch from the
+internet using repository rules. Enables builds to run fully offline.
+
+#### Dynamic execution
+
+An execution strategy that combines both the local and remote execution
+strategy, and through various heuristics, uses the execution result of the
+faster successful strategy. Certain [*actions*](#action) are better executed
+locally (e.g. linking), and others remotely (highly parallelizable compilation),
+so the dynamic execution strategy provides the best possible incremental and
+full build times.
+
+#### Execution phase
+
+The third phase of a build. Executes the [*actions*](#action) from the action
+graph created during the analysis phase. Executed actions invoke executables
+(compilers, scripts) to read and write [*artifacts*](#artifact). *Spawn
+strategies* control how these actions are executed: locally, remotely,
+dynamically, sandboxed, docker, and so on.
+
+#### Execution root
+
+A directory in the workspace’s output base directory where local actions are
+executed in non-sandboxed builds. The directory contents are mostly symlinks of
+input [*artifacts*](#artifact) from the workspace. The execution root also
+contains symlinks to external repositories as other inputs and the `bazel-out`
+directory to store outputs. Prepared during the loading phase by planting a
+"symlink forest" of the directories representing transitive closure of packages
+depended on by the target(s) to build. Accessible with `bazel info
+execution_root` on the command line.
+
+#### File
+
+See [*Artifact*](#artifact).
+
+#### Hermeticity
+
+The property of a build or test where there are no external influences on the
+build or test. This helps to enable deterministic and correct results. This
+usually means disallowing network access to actions, only allowing access to
+declared input [*artifacts*](#artifact), making timestamps and timezones fixed,
+restricting access/resetting environment variables, and using fixed seeds for
+random number generators.
+
+#### Incremental build
+
+A build that reuses the results of earlier builds to reduce build time and
+resource needs. Bazel aims to guarantee correct results for incremental builds.
+This is the opposite of a clean build.
+
+<!-- TODO: #### Install base -->
+
+#### Label
+
+An identifier for a target. A fully qualified label such as
+`//path/to/package:target` consists of `//` to mark the workspace root
+directory, path/to/package as the directory that contains the [*BUILD
+file*](#build-file) declaring the target, and :target as the name of the target
+declared in the aforementioned BUILD file. May also be prefixed with
+`@my_repository//<..>` to indicate that the target is declared in an *external
+repository* named `my_repository`.
+
+#### Loading phase
+
+The first phase of a build. WORKSPACE, BUILD, and .bzl files are parsed to
```

Comment:
Again, take a look at usage of backticks to specify monospaced fonts throughout the document.

Generally, we want to signal typographically whenever a special term is used to indicate a filename, file type, or snippet of command or code that a user might have to enter. So, here, preferentially, we would use `WOKRSPACE`, `BUILD` and `.bzl` to set off the terms, though I imagine this is very inconsistent throughout the doc set.

Reply:
Thanks! I think you may be looking at an older revision of the file in this PR - I've done a pass for the backtick change throughout the document.